packages/core/src/components/SplitButton/SplitButton.tsx

```diff
@@ -10,15 +10,13 @@ import {
   DEFAULT_DIALOG_HIDE_TRIGGER,
   DEFAULT_DIALOG_SHOW_TRIGGER,
   DIALOG_MOVE_BY,
-  EMPTY_ARR,
```

Comment:
Is it still in use or it can be removed from the constants file?

Reply:
Yes it's used in some other places but we will remove it in upcoming prs